publication-commons/src/main/java/no/unit/nva/publication/rightsretention/RightsRetentionsValueFinder.java

```diff
@@ -98,7 +98,7 @@ private boolean isValid(RightsRetentionStrategy rrs) {
     private boolean isAllowedToOverrideRrs(RightsRetentionStrategy rrs,
                                            PublicationPermissionStrategy permissionStrategy) {
         if (nonNull(permissionStrategy) && rrs instanceof OverriddenRightsRetentionStrategy) {
-            return permissionStrategy.isCuratorOnPublication();
+            return permissionStrategy.isPublishingCuratorOnPublication();
         }
```

Comment:
Had to add new method -> isPublishingCurator to differentiate between curator and publishing curator, not sure it is the best solution.